src/PublicProfile/PublicProfile.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import Header from '../Header';
+import Footer from '../Footer';
+import ProfileImage from './profileImage';
+import ProfileDetails from './profileDetails';
+import SkillLabel from './skillLabel';
+import Projects from '../Projects';
+import './PublicProfile.css';
+
+class PublicProfile extends React.Component {
+  render() {
+    return (
+      <div className="container">
+        <div className="header">
+          <Header />
+        </div>
+        <div className="image">
+          <ProfileImage />
+        </div>
+        <div className="details">
+          <ProfileDetails />
+        </div>
+        <div className="skills">
+          <SkillLabel />
+        </div>
+
+        <div className="projects">
+          <h3>Projects</h3>
+          <Projects count={2} />
+        </div>
+        <div className="footer">
+          <Footer />
+        </div>
+      </div>
+    );
+  }
+}
+
+export default PublicProfile;

```

Comment:
instead of line 39, you are going to need to connect it to redux like so: 

```
function mapStateToProps(state: Store) {
  return {
    user: state.user,
    projects: state.projects
  };
}
export default connect(mapStateToProps, { getProjects })(PublicProfile);

```